ouroboros-consensus/test-storage/Test/Ouroboros/Storage/VolatileDB/StateMachine.hs

```diff
@@ -534,7 +534,7 @@ knownLimitation model (At cmd) = case cmd of
         isLimitation (Just slot') slot = slot' >  slot
 
 prop_sequential :: Property
-prop_sequential = withMaxSuccess 1000 $
+prop_sequential = withMaxSuccess 50000 $
```

Comment:
I don't think we should hard-code the number of tests here.

Reply:
I usually forget to remove those.. However do you think it would make sense to have this number on 1000? It is the only test we run for each component, so it would make sense to have more than the standard 100 tests.

Reply:
Nah, this should be done on the command line, not hardcoded in the test. I agree 100 is not usually enough, but we should locally run the test suite with, say, `--quickcheck-tests 10000` from time to time.